public/app/features/alerting/unified/hooks/useAbilities.test.tsx

```diff
@@ -142,20 +141,28 @@ describe('alertmanager abilities', () => {
   });
 });
 
+setupMswServer();
+
+/**
+ * Render the hook result in a component so we can more reliably check that the result has settled
```

Comment:
If I didn't do this, one of the tests could always pass, because it would return `false, false`, and then assume all was ok - but the hook render wasn't triggering an `act` warning due to it changing afterwards. So its safer to test this within a little mock component that renders things based on the hook result. And then this way it also gives a direct example of the hook being consumed